pkg/proto/operator/v1/operator.pb.go

```diff
@@ -33,6 +33,63 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
+// ResourceEventType is the type of event to a resource.
+type ResourceEventType int32
+
+const (
+	// UNKNOWN indicates that the event type is unknown.
+	ResourceEventType_UNKNOWN ResourceEventType = 0
+	// CREATED indicates that the resource has been created.
+	ResourceEventType_CREATED ResourceEventType = 1
+	// UPDATED indicates that the resource has been created or updated.
+	ResourceEventType_UPDATED ResourceEventType = 2
+	// DELETED indicates that the resource has been deleted.
+	ResourceEventType_DELETED ResourceEventType = 3
+)
+
+// Enum value maps for ResourceEventType.
+var (
+	ResourceEventType_name = map[int32]string{
```

Comment:
Could we use enum to skip those convert?

Reply:
Similarly, this is generated code which I didn't write.